.vib/common/goss/templates/check-nginx-libphp.yaml

```diff
@@ -0,0 +1,12 @@
+file:
```

Comment:
As this is intended to be used as a template test, please provide a brief description so as to comply with the rest of the files content:

https://github.com/bitnami/containers/blob/3b4311d5d6202dddf87a61cec655b388321732ef/.vib/common/goss/templates/check-app-version.yaml#L1-L7